pkg/common/aws/session.go

```diff
@@ -15,10 +16,11 @@ import (
 )
 
 type ccsAwsSession struct {
-	session *session.Session
-	iam     *iam.IAM
-	ec2     *ec2.EC2
-	once    sync.Once
+	session   *session.Session
+	accountId string
```

Comment:
Should we rename anything account id to be just account or rename account to be account id. Just thinking to be consistent throughout the code. What do you think?